andela/travel_tool_front
# #162305343 Disable and restore guesthouses

#### What does this PR do?
Gives Travel admin the ability to disable and restore guesthouses.

#### Description of Task to be completed?
* Travel admin should be able to disable guesthouses and view a list of them.
* Travel admin should be able to restore guesthouses.
* Makeup designs where necessary

#### How should this be manually tested?
* Clone the repo ```git clone https://github.com/andela/travel_tool_front.git```
* Change directory with ```cd travel_tool_front```
* Checkout to the branch ```ft-disable-and-restore-guest-houses-162305343```
* Run ```yarn start``` on your terminal to start up the server (also remember to run ```yarn start:dev``` for the back-end)
* Navigate to ```http://travela-local.andela.com:3000``` in your browser
* You need a ```Travel Admin``` privilege.
* Navigate to the Residence/ Manage.
* Select a guest house.
* You should see a ```disable guest house``` option by the top right.
* Click on this and confirm to disable the guest house.
* You can view a list of disabled guesthouses on the ```Residence/ Manage``` page.
* You cannot perform any operation on the disabled guesthouse until it is restored.
* Rooms of disabled guesthouses are also not available when making travel requests.
* On a disabled guesthouse, click ```restore``` button and confirm to restore the guesthouse.



#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[162305343](https://www.pivotaltracker.com/story/show/162305343)

#### Screenshots (if appropriate)
![disablegh](https://user-images.githubusercontent.com/29521319/49597344-80628000-f97c-11e8-89bf-c6f1ff3889c1.gif)
![restoregh](https://user-images.githubusercontent.com/29521319/49597362-8c4e4200-f97c-11e8-964e-3559d513cb45.gif)



#### Questions:
